### PR TITLE
feat: add chinese company

### PR DESCRIPTION
**Pull request by fred7BF**

### CODE REVIEW

**Comment by Shinigami92** [comment]
Do you have a source whhere this is coming from? Any additional context?

File: src/locales/zh_CN/company/name_patterns.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export default ['{{location.city}}{{company.prefix}}{{company.suffix}} '];
```
Comment (ST-DDT) [inline src/locales/zh_CN/company/name_patterns.ts]: This looks wrong to me. A prefix shouldn't be in the middle and there is a space at the end.
Maybe name the file differently?

File: src/locales/zh_CN/company/adjective.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  '管理',
+  '强制',
+  '监控',
+  '多通道',
+  '多边',
+  '多层次',
+  '多层次',
+  '网络',
+  '基于对象',
+  'Open-architected',
```
Comment (ST-DDT) [inline src/locales/zh_CN/company/adjective.ts]: Is this English term here intentional?

**Comment by ST-DDT** [comment]
@fred7BF Was closing this PR intentional?

**Pull request closed without merging**